src/plugins/graph/components/attribute-label.tsx

```diff
@@ -49,12 +49,14 @@ export const AttributeLabel = observer(
 
     const getLabel = useCallback(() => {
       if (useClickHereCue) {
+        if (place === "bottom") return "x";
```

Comment:
I brought this change back because it _is_ the case that the labels should be changed to x and y. Also, clicking on an axis label to link data should not be a feature. Linking will be handled by a yet-to-be-added toolbar button, or by dragging a table to the graph.

I think that means a click here cue is no longer needed in the context of CLUE, but that we want to keep most of the related code for the sake of compatibility between CLUE and CODAP.